core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/gasprice/gasprice.go

```diff
@@ -27,17 +29,18 @@ type UpkeepOffchainConfig struct {
 // CheckGasPrice retrieves the current gas price and compare against the max gas price configured in upkeep's offchain config
 // any errors in offchain config decoding will result in max gas price check disabled
 func CheckGasPrice(ctx context.Context, upkeepId *big.Int, offchainConfigBytes []byte, ge gas.EvmFeeEstimator, lggr logger.Logger) encoding.UpkeepFailureReason {
+	// check for empty offchain config
 	if len(offchainConfigBytes) == 0 {
 		return encoding.UpkeepFailureReasonNone
 	}
 
 	var offchainConfig UpkeepOffchainConfig
 	if err := cbor.ParseDietCBORToStruct(offchainConfigBytes, &offchainConfig); err != nil {
-		lggr.Errorw("failed to parse upkeep offchain config, gas price check is disabled", "upkeepId", upkeepId.String(), "err", err)
+		lggr.Debugw("failed to parse upkeep offchain config, gas price check is disabled", "offchainconfig", hexutil.Encode(offchainConfigBytes), "upkeepId", upkeepId.String(), "err", err)
```

Comment:
should we have these as warn? I don't think we expect these unless the user intentionally makes a mistake?

Reply:
done